fr.openfisca.org/fr.openfisca.org.conf

```diff
@@ -63,6 +63,27 @@ server {
         rewrite ^\/api\/v2(.*) /api/v18$1 redirect;
     }
 
+    set $served_versions '
+    {
+      "v18": {
+        "href": "https://fr.openfisca.org/api/v18/"
+      },
+      "v19": {
+        "href": "https://fr.openfisca.org/api/v19/"
+      },
+      "v20": {
+        "href": "https://fr.openfisca.org/api/v20/"
+      },
+      "v21": {
+        "href": "https://fr.openfisca.org/api/v21/"
+      }
+    }';
```

Comment:
FI it's not possible to _both_ define this in a separate file _and_ return a 300 code